.circleci/config.yml

```diff
@@ -137,6 +123,39 @@ jobs:
       - run:
           name: Run nox on speech
           <<: *anchor_test_python
+  test-ruby:
+    working_directory: /tmp/
+    docker:
+      - image: circleci/ruby:2.4-node
+    steps:
+      - attach_workspace:
+          # Use the workspace created in generate-clients
+          at: workspace
+      - run:
+          name: Install Ruby client dependencies.
+          command: |
+            gem install bundler
+            git clone https://github.com/GoogleCloudPlatform/google-cloud-ruby
+            cd google-cloud-ruby
+            bundle update
```

Comment:
Can you explain this step?

Reply:
Ok, removed. 
(I had copypasting the steps in the Cloud Release Playbook)